bigframes/session/__init__.py

```diff
@@ -307,6 +309,72 @@ def read_gbq(
                 api_name="read_gbq",
             )
 
+    def _filters_to_query(self, query_or_table, filters):
```

Comment:
Seems like a great candidate to write unit tests (string-in-string-out). In fact, at a later point it can go in a sql helper module like `bigframes/ml/sql.py`.

Reply:
Moved to unit test.